demos/PuppeteerSharpPdfDemo/PuppeteerSharpPdfDemo-Local.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp3.1;net471</TargetFrameworks>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
<3

Reply:
Those `using` statements are the spice of life